src/_includes/components/footer.njk

```diff
@@ -15,8 +15,11 @@
           <label for="newsletter-email">Email address</label>
           <div class="field--connected-btn">
             <input id="newsletter-email" type="email" name="email" 
-              placeholder="name@example.com" autocapitalize="off" required />
-            <input type="hidden" name="tags" value="Web Sign up"/>
+              placeholder="name@example.com" autocapitalize="off" 
+              pattern="^[^@]+@[^@]+\.[^@]+$"
+              title="example@gmail.com"
```

Comment:
Wonder if the value of title should be the same as the `placeholder` value? Probably shouldn't use `gmail` for either just to not invoke any certain mail provider.

Reply:
Fixed!